config.toml.example

```diff
@@ -29,6 +29,10 @@
 	mainFacebook = "URY1350"
 	newsTwitter = "URYNews"
 
+	[pageContext.sustainer]
+		name = "Overnight Owen"
```

Comment:
IDK if people want me to change this default to `URY Jukebox` or something.  I mainly put it here as an outdated joke.